validator/validator.bal

```diff
@@ -32,8 +97,10 @@ public isolated function validate(json|anydata data, typedesc<anydata>? targetFH
         anydata|r4:FHIRParseError parsedResult = parser:parse(data, targetFHIRModelType);
 
         if parsedResult is r4:FHIRParseError {
-            return <r4:FHIRValidationError>r4:createFHIRError("FHIR resource validation failed", r4:ERROR, r4:INVALID, parsedResult.message(),
-                                                errorType = r4:VALIDATION_ERROR, cause = parsedResult);
+            log:printDebug("Error is a FHIRParseError");
```

Comment:
Let's log the error as well. Also reword the message like "FHIR parsing failed."